docs/development/ansible_playbook_bundles.md

```diff
@@ -0,0 +1,112 @@
+# FeedHenry Ansible Playbook Bundle Development Guide
+
+The document is documenting best practises for the FeedHenry APB development! Some general information about the Ansible Service Broker (ASB) & Ansbile Playbook Bundles (APB) can be found[here](https://docs.openshift.com/container-platform/3.6/architecture/service_catalog/ansible_service_broker.html)
+
+## Installation 
+
+The `apb` CLI can be installed locally, _however_ it is highly recommended using it wrapped in a Linux container. Below is an example on how to create an alias for the _wrapped_`apb` CLI:
+
+```bash
+alias apb='docker run --rm --privileged -v $PWD:/mnt -v $HOME/.kube:/.kube -v /var/run/docker.sock:/var/run/docker.sock -u $UID docker.io/feedhenry/apb'
+```
+
+
+## Creating an APB
+
+Using the `apb init` command you can create a template for your APB, like:
+
+```
+apb init DOCKERORG/NAME_OF_YOUR_STUFF-apb 
+
+```
+
+_NOTE:_ It's a good practise to have the APB end with `-apb`!
+
+
+### Create a Makefile
+
+Once you have made the changes for (de)provisioning or (un)binding, you _could_ use `apb prepare` and `apb build` to run the build. Using a `Makefile` is a much better idea. Currently our FeedHenry APBs do have a `Makefile`, like [here](https://raw.githubusercontent.com/feedhenry/fh-sync-server-apb/master/Makefile):
+
+```
+DOCKERHOST = docker.io
+DOCKERORG = feedhenry
+USER=$(shell id -u)
+PWS=$(shell pwd)
+build_and_push: apb_build docker_push
+
+.PHONY: apb_build
+apb_build:
+	docker run --rm -u $(USER) -v $(PWD):/mnt:z feedhenry/apb prepare
```

Comment:
this needs indentation to be a valid makefile.

Reply:
I wonder if we could make a PR to the APB tool, to generate a sample makefile in the init?

Reply:
I created a PR for this: https://github.com/ansibleplaybookbundle/ansible-playbook-bundle/pull/149